proposed/net-app-launch.md

```diff
@@ -0,0 +1,29 @@
+
+# Launching .NET Apps
+
+This document covers recommendations for launching .NET applications. There are two main form factors .NET app deployment:
+
+* Self-contained
+* Framework-dependent
+
+Self-contained apps are the simplest. They have a platform-native entry executable and contain all files needed for their execution. They match the native platform standard for launching executables. They do not support configuration beyond the native platform convention.
+
+Framework-dependent apps do not contain a framework themselves and instead need a separate framework located somewhere on the machine. This is the most complicated scenario with the most configuration options.
+
+## Framework-dependent apps
+
+There are two ways to run framework-dependent apps: through the "apphost" launcher and via `dotnet app.dll`. Whenever possible, it's recommended to use the apphost. There are a number of advantages to using the apphost:
+
+* Executables appear like standard native platform executables.
+* Executable names are preserved in the command line, meaning apps can be easily recognized based on their names.
+* Because the apphost is a native binary, native assets like manifests can be attached to them.
```

Comment:
```suggestion
* Because the apphost is a native binary, native assets like manifests can be attached to them.
* Apphost has available [low-level security mitigations](https://github.com/dotnet/designs/blob/main/accepted/2021/runtime-security-mitigations.md) applied by default that makes it more secure. Mitigations applied to `dotnet` are the lowest common denominator of all supported runtimes.
```